### DISCUSSION TITLE
How to render headmatter data in custom layout? Missing docs or feature, not sure which one.

### DISCUSSION THREAD

**Discussion by barrymichaeldoyle** [slidevjs/slidev#1927] [Q&A]
**Is your feature request related to a problem? Please describe.**
I would like to use headmatter data inside my custom layout but there isn't any documentation on how this can be done.

**Describe the solution you'd like**
If I have a `cover` layout. How can I render a title in the template?

`example.md`
```
---
layout: cover
title: My Title
---
```

`/layouts/cover.vue`
```
<template>
   <div class="slidev-layout cover"
     <h1>{/* How do I render "My Title" here? */}</h1>
     <slot />
   </div>
</template>
```

**Describe alternatives you've considered**
The way I currently do it is by manually styling `h1` tags inside the `cover` css class.

**Comment by kermanx** [comment]
Try `$frontmatter.title`?

Here are some docs: https://sli.dev/guide/global-context#frontmatter
